src/training/train.py

```diff
@@ -69,20 +79,60 @@ def train_one_epoch(model, data, epoch, optimizer, scaler, scheduler, args, tb_w
     end = time.time()
     for i, batch in enumerate(dataloader):
         step = num_batches_per_epoch * epoch + i
-        
+
         if not args.skip_scheduler:
             scheduler(step)
 
-        images, texts = batch
+        images, texts, text_input_masks = batch['image'], batch['text'], batch['text_input_mask']
```

Comment:
This probably can't land as it is. We need a batch level abstraction which moves all of the elements in the batch directly to GPU when `.to('cuda')` is done. Otherwise, these kind of changes will bloat the trainer.